classes/class3/catHW/public/rand.js

```diff
@@ -0,0 +1,21 @@
+function randInt(length) {
```

Comment:
Absolutely does NOT go in public. No code that is used by the server itself should go in public. Only put things there that is used by the client.